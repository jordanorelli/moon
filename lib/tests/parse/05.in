# this was original a variable test case, but I'm changing it to be a bare
# string / quoted string test.  The variable test will come after; variables
# are broken right now.

quoted_string: "this is a quoted string"
bare_string: this is a bare string
name with space: that name has spaces in it, and you know what? that's ok!
