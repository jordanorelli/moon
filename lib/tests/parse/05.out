root:
  assign:
    name:
      quoted_string
    value:
      string:
        this is a quoted string
  assign:
    name:
      bare_string
    value:
      string:
        this is a bare string
  assign:
    name:
      name with space
    value:
      string:
        that name has spaces in it, and you know what? that's ok!
